lib/events.js

```diff
@@ -961,7 +961,7 @@ async function once(emitter, name, options = kEmptyObject) {
   const signal = options?.signal;
```

Comment:
(non-blocking)
```suggestion
  const signal = options.signal;
```
Because we already did `validateObject`